config/initializers/sidekiq.rb

```diff
@@ -2,25 +2,24 @@
 require 'sidekiq-scheduler'
 
 is_production = (ENV['IS_PRODUCTION'] || "false").downcase == 'true'
-redis_url = ENV.fetch('REDISCLOUD_URL') if is_production
+redis_url = ENV.fetch('REDISCLOUD_URL')
 
 Sidekiq.configure_server do |config|
-  if is_production
-    config.redis = { url: redis_url }
-  end
+  config.redis = { url: redis_url }
 
   config.on(:startup) do
     # load production-specific schedule config
     if is_production
-      Sidekiq.schedule = YAML.load_file(File.expand_path('../../sidekiq_production.yml', __FILE__))
-      SidekiqScheduler::Scheduler.instance.reload_schedule!
+      Sidekiq.schedule = YAML.load_file(File.expand_path('../../sidekiq_schedule_production.yml', __FILE__))
+    else
+      Sidekiq.schedule = YAML.load_file(File.expand_path('../../sidekiq_schedule_default.yml', __FILE__))
```

Comment:
That was my initial instinct but the name was kinda long so I had gone with 'other' :sweat_smile:

Reply:
Sure, I think I felt it was better to stick with Rails naming conventions but probably not here. In most cases we seem to extend `default` config rather than override it so I can see how this could be easily missed.